### PR TITLE
web/satellite Billing screen bug fixes

### PR DESCRIPTION
**Pull request by cl-mitch**
What: Changed the Billing History tab to only display Stripe payments, and addressed the following tickets:
https://github.com/storj/storj/issues/5028
https://github.com/storj/storj/issues/5027
https://github.com/storj/storj/issues/5026
https://github.com/storj/storj/issues/5021

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

**Comment by nerdatwork** [comment]
Additionally, please review my observations as they are part of billing.
https://forum.storj.io/t/release-preparation-v1-60/19182/5?u=nerdatwork

**Comment by cl-mitch** [comment]
> Additionally, please review my observations as they are part of billing. https://forum.storj.io/t/release-preparation-v1-60/19182/5?u=nerdatwork

Hi @nerdatwork, This PR (and a few other recent ones) should address the issues that you are seeing, but they won't be reflected in the QA Satellite until they are merged into main and a new build is released for QA. Is that where you are seeing these issues today, or is it locally with these changes pulled?

**Comment by nerdatwork** [comment]
Hi @cl-mitch, I saw them on QA satellite when Andrii asked to give feedback.

**Comment by cl-mitch** [comment]
> Hi @cl-mitch, I saw them on QA satellite when Andrii asked to give feedback.

@nerdatwork Thanks for the quick response! Not entirely sure when the QA Satellite will be updated, but when it is it will reflect some or all of the fixes.

**Comment by cl-mitch** [comment]
run jenkins

File: web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue [hunk 1]
```diff
@@ -28,8 +28,15 @@ import { PaymentsHistoryItem } from '@/types/payments';
 })
 
 export default class BillingArea extends Vue {
+
+    public mounted(): void {
+        console.log(this.historyItems)
```
Comment (mobyvb) [inline web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue:33]: remove console.log

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue [hunk 2]
```diff
@@ -4,31 +4,31 @@
...
                     <div class="coupon-area__container__existing-coupons__status-container">
-                        <span class="coupon-area__container__existing-coupons__discount-top-container active">
+                        <span class="coupon-area__container__existing-coupons__status-container__status active-status">
                             Active
                         </span>
                     </div>
                     <div class="coupon-area__container__existing-coupons__discount-black-container">
                         <span class="coupon-area__container__existing-coupons__discount-black-container__discount">
-                            ${{ coupon.getDescription() }} off
+                            {{ coupon.getDescription().slice(0, coupon.getDescription().indexOf(' ')) }} off
```
Comment (mobyvb) [inline web/satellite/src/components/account/billing/billingTabs/Coupons.vue:26]: What usually appears after the space in the coupon description?

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue [hunk 3]
```diff
@@ -85,6 +85,8 @@ export default class CouponArea extends Vue {
         } catch (error) {
             await this.$notify.error(error.message);
         }
+
+        console.log(this.coupon)
```
Comment (mobyvb) [inline web/satellite/src/components/account/billing/billingTabs/Coupons.vue:89]: remove console.log

File: web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue (hunk 1 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue:33]: removed

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 3 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/Coupons.vue:89]: removed

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 2 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/Coupons.vue:26]: There is text returned that is beyond the scope of the figma.

File: web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue (hunk 1 above)
Comment (VitaliiShpital) [inline web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue:33]: remove console.log

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 3 above)
Comment (VitaliiShpital) [inline web/satellite/src/components/account/billing/billingTabs/Coupons.vue:89]: remove console.log

File: web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue (hunk 1 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/BillingHistory.vue]: removed

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 3 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/Coupons.vue]: removed

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue [hunk 4]
```diff
@@ -82,8 +80,10 @@ export default class CouponArea extends Vue {
         try {
             await this.$store.dispatch(PAYMENTS_ACTIONS.GET_COUPON);
             this.isCouponFetching = false;
+            console.log(this.coupon)
```
Comment (mobyvb) [inline web/satellite/src/components/account/billing/billingTabs/Coupons.vue:83]: remove console.log

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue [hunk 5]
```diff
@@ -109,81 +109,73 @@ export default class CouponArea extends Vue {
...
+            return this.expiration;
         }
     }
+
 }
 </script>
 
 <style scoped lang="scss">
-    .active-discount {
+    .Active-discount {
```
Comment (mobyvb) [inline web/satellite/src/components/account/billing/billingTabs/Coupons.vue:164]: what's the reason for capitalizing the class name? I feel like we should keep it lowercase unless there's a good reason not to.

**Review by mobyvb — COMMENTED** [review]
It looks good to me other than the minor comments I left

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 5 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/Coupons.vue]: resolved

File: web/satellite/src/components/account/billing/billingTabs/Coupons.vue (hunk 4 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/billingTabs/Coupons.vue]: resolved

**Review by mobyvb — APPROVED** [review]

**Review by storj-antonio — APPROVED** [review]
LGTM, tested locally using storj-up.

**Review by wilfred-asomanii — APPROVED** [review]

**Pull request merged**